src/annotator/config/index.js

```diff
@@ -162,8 +198,33 @@ export function getConfig(appContext = 'annotator', window_ = window) {
   let filteredKeys = configurationKeys(appContext);
   filteredKeys.forEach(name => {
     const configDef = configDefinitions[name];
+    const hasDefault = configDef.defaultValue !== undefined; // A default could be null
+    const isURLFromBrowserExtension = isBrowserExtension(
+      urlFromLinkTag(window_, 'sidebar', 'html')
+    );
+
+    // Only allow certain values in the browser extension context
+    if (!configDef.allowInBrowserExt && isURLFromBrowserExtension) {
+      // If the value is not allowed here, then set to the default if provided, otherwise ignore
+      // the key:value pair
```

Comment:
IMO this comment is a bit redundant, as the code is very straightforward and the naming makes it clear what is going on.